### DISCUSSION TITLE
installing the ord wallet on macbook

### DISCUSSION THREAD

**Discussion by bonjwa67** [ordinals/ord#1579] [Q&A]
henlo are there any guides on how to install the ord wallet and run a node on a macbook m1 please

**Comment by raphjaph** [comment]
https://docs.ordinals.com/guides/inscriptions.html

**Comment by joeatang** [comment]
I just learned that Bitcoin Core Client does not automatically come with 'Bitcoind' or 'Bitcoin-Cli' im currently going through a process that should act as a solution and if it does i will share here.
